src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -185,15 +185,14 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    * applicable) once initialized.
    *
    * The behavior of calling initialize() on an already initialized
-   * Manager should be a no-op, but if an error was raised
-   * previously, then initialization will be re-attempted.
+   * Manager is to re-initialize the manager with any updated settings
```

Comment:
[plan] Same problem in `ManagerInterface`

Reply:
Was in two minds whether to just fix it as part of this issue, but instead made https://github.com/OpenAssetIO/OpenAssetIO/issues/1090 as a follow-up.